ci-based/settings.py

```diff
@@ -0,0 +1,32 @@
+# If you want custom worker name
+# NAME = 'worker-1024'
+
+# If you want to use a dictConfig
+# <https://docs.python.org/3/library/logging.config.html#logging-config-dictschema>
+# for more complex/consistent logging requirements.
+DICT_CONFIG = {
```

Comment:
What is this settings file used for? Could it be merged with the previous config file?

Reply:
It's the configuration file of the `rq worker`. I've move this part into config.yml and renamed the file to `rq_worker_settings.py`.